prowler/providers/aws/services/rds/rds_instance_security_group_event_subscription/rds_instance_security_group_event_subscription.py

```diff
@@ -0,0 +1,34 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.rds.rds_client import rds_client
+
+
+class rds_instance_security_group_event_subscription(Check):
+    def execute(self):
+        findings = []
+        report = Check_Report_AWS(self.metadata())
+        report.status = "FAIL"
+        report.check_metadata.Severity = "medium"
+        report.status_extended = 'RDS security group event categories of "configuration change" and "failure" are not subscribed.'
+        report.region = rds_client.region
```

Comment:
This region is the global/default one, we need to use the one that comes from the regional clients.

Reply:
Corrected in latest commit to use the ```db_instances[0].region```